api/views.py

```diff
@@ -6,8 +6,8 @@
 from rest_framework import viewsets
 
 from hunts.models import Hunt
-from puzzles.models import Puzzle, PuzzleModelError
-from .serializers import HuntSerializer, PuzzleSerializer
+from puzzles.models import Puzzle, PuzzleModelError, PuzzleTag, is_ancestor
```

Comment:
Probably beyond the scope of this PR, but shouldn't we make `is_ancestor` a method of Puzzle? Seems like `puzzle_object.is_ancestor_of(other_puzzle_object)` would make more sense.

Reply:
I agree, good suggestion.